In [1]:
%reload_ext autoreload
%autoreload 2

In [8]:
import pandas as pd
import altair as alt

# Fifty Years of Technocracy. International Migration Management as an Evolving Discourse Coalition.

- Marijke van Faassen - Huygens ING (marijke.van.faassen@huygens.knaw.nl)

- Rik Hoekstra - Huygens ING (rik.hoekstra@di.huc.knaw.nl)

- Marijn Koolen - Huygens ING (marijn.koolen@di.huc.knaw.nl)

## Introduction

In 1963 the American demographer and sociologist William Petersen wrote a review article on several books on refugee studies. Except for some studies on the German refugee problem, at that time still an important issue due to the millions of post-World War II ethnic and non-ethnic German expellees from Central and East Europe on German soil, Petersen also reviewed the first issue of a new journal called International Migration: Quarterly review on the role of migratory movements in the contemporary world (IM). The journal was ‘sponsored’ by the so called Intergovernmental Committee for European Migration (ICEM) and was in fact a merger between two journals: ICEM’s own journal Migration and the bulletin of the so called Research Group for European Migration Problems (REMP), the REMP-bulletin. Petersen praised the high scientific quality of one of the REMP-monographs he also reviewed, but he was extremely critical about this first journal issue. He not just saw the efforts of the Intergovernmental Committee for European Migration to assist migrants to move from Europe as a mere duplication of the purposes of the in 1952 established UN High Commissioner for Refugees, he also questioned the character of the newly established journal: ‘If the first number of International Migration is a good indication, the shift from the REMP Bulletin has been one from a scholarly journal to a self-aggrandizing house organ. […] Most of the articles are by agency officials rather than scholars.’ (Petersen 1963, 419-421).

The Intergovernmental Committee for European Migration is nowadays better known under its current name International Organisation for Migration (IOM). It is one of the bigger players in the field of regulating global migration, albeit not an undisputed one. Currently, in particular IOM’s ‘voluntary’ return programs and IOM’s participation in Australia’s refugee policy in the Pacific are subject to criticism (e.g. Andrijasevic and Walters 2010; Fleay and Hoffman 2014). Since the establishment of the Provisional Intergovernmental Committee for the Migration from Europe (PICMME), the predecessor of both ICEM and IOM, scholars have been struggling to characterize this organization.

Interestingly enough, in 2018, the German geographer and critical IOM-expert Martin Geiger made comparable observations as Petersen did 55 years before that IOM strategically time and again positions itself by actively disseminating a self representation (via its website), that does not necessarily coincide with its behind-the-scenes-character or daily practices. The scholarly debate on international migration has been characterized over the past 30 years by a focus on globalization and a paradigm shift to the study of migration management. The key actors in migration management are the nation states and also the international meetings and organizations consisting of trained experts. They have become more and more the subject of current discourses around the legitimacy of technocratic governance. Geiger and Koch (2018) conceptualize the ‘intergovernmental’ IOM as a ‘world organization’ as IOM has relations with both state and non-state actors, such as NGOs or civil society. This, they argue, makes it possible to analyze which actors participate in the decision-making process, how they participate, and with what authority. Although they start to describe IOM’s history from the start in 1951, their analysis of the early years and especially the 1960s remains fragmentary. Their jump to the mid 1980s leaves ICEM/IOMs historical roots understudied and especially its apparent relation with the expert group REMP, thus leaving ICEM/IOMs  technocratic character (p.39) unexplained.

In this article we will historicize the concept of migration management and unravel the connections between - allegedly technocratic - experts in REMP and ICEM/IOM. We do this by discussing the concept of ‘world organization’ used by Geiger and by operationalizing the concept of discourse coalitions, that is constellations between (academic) experts and actors in the political system who share ideas and  interact, thus strengthening each other's arguments. This will enable us to specify our main research question and to pose the additional sub questions necessary to explore digital methods to lay bare networks of technocrats shaping migration management interactively with national governments. 

## ICEM and the conceptual challenges

As successor of the temporary UN International Refugee Organization (IRO) in 1951, ICEM  framed itself as a  purely technical post-war refugee transport organization, but it rather was an organization that in the first decennia of its existence focused especially on stimulating migration of labour surpluses from Europe and had highly political aims that were born from a combination of the post war  ‘one world’ ideology and Cold War reality (Van Faassen 2014, Ch.3; Parsanoglu 2015). Its twofold objectives and its non-permanent status till the end of the 1980s outside the United Nations family was probably the cause that it remained understudied by both refugee studies and international relations and migration studies for almost forty years. This changed when the scholarly debate on international migration started to concentrate on the study of migration management and researchers discovered the new global ambitions of ICEM in this field. These ambitions found their expression in a new constitution and two name changes, first the skipping of the regional ‘Europe-part’, resulting in ICM in 1980, followed by the shift from ‘intergovernmental committee’ to the more inclusive sounding ‘International organization’ IOM in 1987 when the organization also  obtained a permanent  status, which suggested a governance system in which not only states but also other non-state actors play a role (Perruchoud 1987). Since then, a lot of critical studies on ICEM/IOM have been published. Although IOM’s genesis and early history of the 1950s is well known by now, most of them pay little or no attention  to the in-between decades,  the 1960s-1980s, and the developments in the research that was ‘sponsored’ by the organization. In this way,  scholars neglected the merger of the journals of ICEM and the till now further unnoticed  REMP research group, as well as the early critical notes on the character of ICEM and its self representation for a wider public, that William Petersen put his finger on in his review. 


In their article, Geiger and Koch (2018, 27) propose to conceptualize IOM as a ‘world organization’, in other words as an International Organization embedded in world society with relations with a multitude of different actors. With this new concept they follow the so-called organizational turn in the field of International Relation studies that propose to not only study ‘states’ but to broaden the focus to the organizational character of the actual organizations as well, in order to solve the lack of a firm and common theoretical framework in the critical IOM studies thus far and to get a better understanding of IOM by bringing some of the pieces of the IOM-puzzle together. The four dimensions they suggest to explore - IOM’s semantic self-description and relations to the world; its inner world (structures, processes and participation in decision making); its environmental or external relations and its role in generating world order - are without doubt suitable for a more inclusive analysis of IOM and its interorganizational networks.

However, given their subsequent analysis of IOM’s historical roots (30-32) the concept is too static without a historical dimension and does not address the issues of its genesis or the dynamics of interaction. By historicizing IOM we move away from the aggregate level of the organizational actors and break them down in their constituent historical (human) actors, their contacts and (policy) arguments. This allows us for instance to question the assumption that ‘meaningful inter-state discussions and dialogues between states and international organizations on migration are a post-Cold war phenomenon (starting 1989-90) with the exception of the European Community’ (31). In previous research we have shown by following internal debates between ICEM key actors that ICEM in its earlier decades already facilitated informal multilateral discussions outside Europe on international migration - including consultations with local civil society actors (Van Faassen 2014, esp. 308-311). The more or less finalist approach also leads to inaccuracies in the accusation that IOM is ‘catering exclusively to the restrictive interests of Global North receiving states’ (Georgi 2010, as cited in Geiger and Koch 2018, 34). This overlooks the fact that important Global North member states like the Federated Republic of (West-)Germany, the Netherlands, Italy, Austria, Greece and Spain in the 1950s and 1960s still considered themselves primarily as sending states (Steinert 1995, Freund 2004, Van Faassen 2014, Parsanoglu 2015). As their concept of ‘world organization‘ is also meant to contribute to the emerging field of political sociology, we therefore combine it in this article with the concept of so-called discourse-coalitions, with reference to the argumentative turn in the field of the political sciences (Fischer and Forester 1993).

Dutch political scientist Maarten Hajer (1995) describes a discourse-coalition in short as ‘ a group of actors who share a social construct’ and points out that discourse formation takes place on many different levels and in many different localities and that the discursive elements (narratives or storylines) have an important organizational potential, because they have an agenda setting purpose. One of the benefits of the concept according to Hajer is that it enables analysis of strategic action in the context of sociohistoric discourses and institutional practices and especially offers a tool to analyse specific issues. Below, we will demonstrate that this is the case for ‘migration management’. German historian and sociologist Raphael Lutz (2012) uses the discourse-coalitions concept to analyse the embedding of social sciences in Western Societies and especially in politics. In his view discourse coalitions can be seen as constellations at any given time in which social scientists (experts) develop ideas ‘that strengthen the arguments of a group of actors in the political system, whose policies might, in turn, support the standing of these scientists in academia’ (paraphrase by Brückweh et. al, 8). This is particularly important, as we demonstrated in our previous research that REMP was constituted and based in the Netherlands on the initiative of Dutch demographers and sociologists, who had firm connections to the Dutch Government Commissioner for Emigration Bas Haveman (1908-1979). Haveman led ICEM between 1961-1969 as the only non-American Director-General till 2018 (Van Faassen 2014, 2017; Van Faassen and Hoekstra 2017). Furthermore, Lutz offers a typology consisting of varying configurations of the ‘building blocks’ of such a discourse coalition in general, that is the ideas or metaphors, the experts, the clients, the techniques or technologies used and the institutions. Different configurations that were dominant in specific periods between 1880-1980. This framework is helpful for analysing the supposedly but still insufficiently explained technocratic character of ICEM/IOM (Geiger and Koch 2018, 39) as we will demonstrate.

Combining both the concepts of World Organization and discourse coalitions thus offers us the opportunity to analyse the apparent relation between the almost unknown Research Group for European Migration Problems (REMP) and ICEM/IOM. Given the merger of their journals, REMP was seemingly a part of the ‘external environment’ of ICEM/IOM. But can REMP also be seen as a discourse coalition originating in the Netherlands in the early 1950s?  In this article we want to substantiate a mutual influencing of REMP-experts and ICEM-political or administrative actors over the years through discursive elements and state that it formed the nexus between science and politics in what nowadays is called international ‘migration management’. We also will substantiate and explain how  this supposed discourse coalition developed through time.

We chose a mixed methods approach of close reading and distant reading using digital techniques. For digital analysis we constructed datasets based on both analogue information from archive collections and REMP-publications and long-running journals like International Migration with metadata and title content publicly available online. **Writing this article for a three-layered online Journal on Digital History gave us an excellent opportunity to use and test the data scopes concept we developed in previous research (Hoekstra and Koolen 2018). The idea of this concept is to facilitate doing and communicating on digital research and its sources by describing the iterative steps that have to be taken to study them, by accounting for the decisions taken and make replication possible**. In the following sections we first elaborate on the historical roots of REMP, ICEM and the Dutch connection and operationalize the concept of discourse coalition for this specific case to show where technocracy comes in. This gives us a clue to formulate sub questions necessary to select and compose our datasets. Following this, we present our digital sources, discuss missing links in some of them and present the methodological translation we had to make to answer our main research questions. Finally we analyse and explain our results and address the robustness of our choices.

## Dutchs connections to REMP and ICEM

Technocracy tends to present itself as ‘above politics’, but in the last decades scholars have been actively demystifying the technocratic conception of policy analysis as neutral ‘science’, thus bringing to the surface the politics that is inherent in it. (Fischer 1995; Nord 2010; Habermas 2015). Technocracy is often defined as controlling or replacing democratic deliberation, which is usually based on finding a compromise between conflicting interests,  with a more neutral sounding science-based governance discourse where political issues are more or less transformed into undisputed, technically defined ends that can be pursued through administrative means. Although Lutz (2012) does not focus on ‘technocracy’ as such, he argues that discourse coalitions originate in the desire to make scientifically informed policy. Therefore, his suggestion to study the impact of ‘expertise’ by a combined analysis of both the networks of actors from different backgrounds in academia, administration and political parties and the networks of discourses they produce is in fact a helpful tool to unravel the technocratic character of certain issues in politics, because it lays bare how the concepts and arguments of the social and human sciences got linked to the political domain and vice-versa. Thus, Lutz argues, using the concept of discourse coalitions requires elementary prosopographic evidence (whenever available), but also a keen eye for the actual genesis of possible expert groups. Below, we first discuss the founding years of the Dutch emigration governance system, REMP and especially the predecessor of ICEM, the Provisional Intergovernmental Committee for the Movement of Migrants from Europe (PICMME). By using Lutz’ typology of the configuration of discourses and metaphors used in those periods we study the initial connections between social science expertise and international migration. This analysis gives rise to conclusions and questions about the relation between REMP and ICEM and to sub questions to select our sources and construct our datasets to answer our main research questions. 

The merger in 1961 between the REMP-bulletin and ICEMs magazine that William Petersen reviewed coincided with the American and Australian lobby for the Dutchman Bas Haveman as the new Director-General of ICEM. Haveman was a social-democratic chemical engineer from Delft Polytechnic University who also studied law at Leiden University and had been appointed Dutch Government Commissioner for Emigration in November 1950. But both Haveman and REMP already had a history, that is important to understand the genesis of REMP. The roots of the Dutch migration policies lie at the end of the 19th and the first half of the 20th century, periods that are characterized by discourses on ‘social reform’, targeted on the ‘working poor’  (1880-1910) and ‘social engineering’ (1910-1940). According to Lutz (2012), especially after World War I the interventions were grounded in social science research and were based on arguments concerning ‘fear of degeneration’ and - more positively formulated - ‘national recovery’, with discourses around ‘demography’ , ‘community’ or ‘eugenics'. The Dutch migration policies fit well into Lutz’ characterization of the impact of expert-influence in the Western societies in general. 

Although today’s polarized debates on economic migration seem to suggest otherwise, for more than a century states have considered (international) migration – and thus the regulation of it - as a sound instrument for the (re)allocation of labour. In 1913, the Netherlands Association for Resettlement (landverhuizing)  was founded, as a hybrid private-public organization. Amongst its initiators were the liberal Minister of Agriculture Willem Treub and an engineer from Delft Polytechnic University, Isaac Pieter de Vooys. Both were still working on their final report by the State Commission on Unemployment, in which they explored internal and international migration as possible solutions (Van Faassen 2014).  De Vooys was also part of an intellectual movement that has been described as synthetic technocracy, that was looking for  a synthesis that would bridge ideological differences and thus stop the fragmentation of Dutch society which was divided along religious lines (so called pillarization) at that time, and that paralyzed Dutch political decision-making in their view. 

During the interbellum their quest became increasingly connected with proposals for educational reform (Baneke 2011, 91)  and the movement began to coincide with the educational program of the socially engaged Professor Sebald Steinmetz. He developed a new research style called ‘sociography’ (the predecessor of sociology) focussing on empirical research on Dutch social groups and communities in relation to different people internationally. One of his students initiated the so-called People’s High School Movement in the Netherlands. This had comparable aims to the technocrats and attracted other Dutch  sociographers, like Evert Hofstee and Pieter Bouman who later also played a role in REMP. During the world wide economic crisis of  the 1930s the Friesland High School organized combined employment and emigration courses, thus creating the possibilities for demography-based politics. One of the teachers was Bas Haveman (the later Government commisioner), who had previously been the secretary of the “Stichting Nederlands Volkskracht” (Dutch People's Power Foundation) that aimed at moral and physical  training of unemployed youth, to prevent ‘moral decay’ and to stimulate ‘community spirit’ (gemeenschapszin) especially to bridge the then perceived gaps in mentality between city and countryside (Van Faassen, 2014, Van Faassen and Hoekstra 2017).

After World War II social engineering did not disappear, but emanating in the USA and the UK, it became instrumental for establishing a peaceful and above all a ‘planned modernization’, in which welfare states could be extended, according to Lutz (2012). Metaphors for this modernization discourse were ‘assimilation’ and 'adaptation’. Again the Netherlands were no exception to the general picture. Already during the war the Dutch government had started to investigate possibilities for a so-called active migration policy, emanating from a Keynesian aspiration for full employment. It based its post-World-War II socioeconomic policy on two cornerstones. In order to prevent structural unemployment, industrialization was to be encouraged. In addition, large-scale family emigration to overseas destinations was stimulated and facilitated. This not only enabled the Dutch government to manipulate the supply side of the labour market, but also to avoid the at that time unwanted demographic solutions like birth control in the still religious Netherlands, which had the highest increase in population forecast till the 1960s, due to its sky high birth rates (UN Population forecast 1951; Van Faassen 2014, Van Faassen and Hoekstra 2017). Not surprisingly, when Haveman was appointed Government Commissioner for Emigration in 1950 he believed that migration policy had to be science-based and planning was only possible if he had an insight into the migration potential of the Dutch population.

Therefore he established close connections between his Commissioner’s Office of Emigration, based at the ministry of Social Affairs and social scientists and demographers, most of them from his former High School networks. He also established new acquaintances. Amongst them was the German political economist and demographer Günther Beijer, who fled to the Netherlands in 1933. Beijer was praised in Petersen’s review as the one who ‘created and sustained the Research Group for European Migration Problems’ - he was editor of the REMP-publication series till his death in 1983 - and who ‘achieved an enviable reputation among demographers’ (Petersen 1963, 420). According to his biographer, Beijer was also known for his skills in ‘facilitating a dialogue between politicians and demographers’ (Van de Kaa 1983, 3). Haveman met Günther Beijer when they both were introduced to the American Delegation during its stopover in The Hague, just before the start of the founding Conference of PICMME in Brussels, Belgium in november 1951, and only a few months before REMP was established in March 1952 (IISG, collection Beijer, inv.nr. 33).

In his latest analysis of IOM, Geiger (2018) does not connect the founding of IOM’s predecessor PICMME with the history of the International Labour organization (ILO), the International Refugee Organization (IRO) and the United Nations High Commissioner on Refugees (UNHCR), although such an analysis was part of older ILO-studies (Alcock 1971). This interconnectedness is of importance for recognizing some of the technocratic arguments that accompanied the establishment of PICMME/ICEM, to better understand its longtime existence outside the UN as well as the Dutch influence in ICEM. The Netherlands took an active part in the international emigration debate after World War II, in which every country had its own agenda, while humanitarian and economic arguments concerning manpower became inseparably linked to arguments on collective security due to quickly deteriorating East-West relations.  Another element of Haveman’s job was strengthening international contacts. He acquired his first international experience as a social-economic expert in the Dutch delegation to the United Nations (UN) as a member of the Special Committee on Refugees and Displaced Persons, that  set up the International Refugee Organization (IRO) in 1946, a temporary UN-agency in which the Eastern (communist) European countries decided not to participate. In this role Haveman worked closely with United States (US) delegate George L. Warren, who was the State Department’s advisor on refugees and later became the head of the American delegation to ICEM till the mid-1960s (Van Faassen 2014, 2017).

In the early 1950s, it was unclear which international institutions would regulate and be able to finance refugee and migrant flows. By the time the mandate of the IRO had expired at  the end of the 1940s the refugee problem had still not been resolved. The UN subsequently decided to establish the Office of the UN High Commissioner for Refugees (UNHCR), but with only a small administrative budget and at first without an operational mandate  (Salomon 1991, Zieck 1997). Due to Cold War considerations and the ‘white’ migration policies of Canada and Australia, the Western countries were also looking for a continuation of the IRO services outside of the UN system with its communist and ‘non-white’ member states. The International Labour Organization tried to enhance its position by arguing that migration traditionally was a ‘labour-’ issue and therefore should ressort under ILO. In 1951, Haveman convinced the Dutch government that ILO should not emerge as winner of the battle over the IRO’s material legacy (the fleet, personnel and an administrative budget augmented by a US bonus of ten million dollars), even though the current social-democratic Dutch government initially favoured the ILO option because of the tripartite composition of its governing and plenary bodies with union and employers delegates. Haveman informed the Dutch government that during his travels to Canada and Australia, he had experienced resistance to ILO interference due to the non-Western input that was difficult to combine with the de facto white migration policy of both countries. Haveman had also been in touch with Warren and learned that the US Congress would not finance any operational ILO migration work due to strong anti-ILO forces in US domestic politics. As a result the Netherlands supported the solution,  of allocating the IRO legacy for one year to the PICCME, which was instigated and presented by US Congressman Francis E. Walter at the conference in Brussels in November 1951.  

PICMME presented itself as purely a transport organization for refugees and displaced persons, temporarily offering technical solutions for what was publicly stressed as a humanitarian problem. However, internally the member states had agreed that its main function was to encourage and facilitate the economic migration of labour surplus from a disrupted Europe, giving  this ‘depoliticized’ agreement in fact a highly political background. The preference for bilateral contacts on migration remained, while at the same time ICEM facilitated multilateral discussions without asking for too much collective commitment from the member states because of its temporary status outside the UN-system. Therefore, in the long run it could even function as a safety valve for an integrating Europe: the moment the free movement of people within Europe actually would be regulated  - in which Germany and the Netherlands would likely become the ‘receiving’ countries - this inflow could be compensated by facilitating the overseas emigration of their own nationals using PICMME / ICEM facilities  (Van Faassen 2014, Parsanoglu 2015). One of the advantages of this ‘light’ construction was that PICMME’s and from 1953 onwards  ICEM’s Executive Board, on which the Netherlands had an almost permanent seat until the 1970s, was only accountable to its own Council and thus could stay under the radar of the national parliaments. 

The day in november 1951 that Günther Beijer met Haveman at the US embassy in the Hague, Beijer was invited to inform Congressman Walter and the other American delegates about the plans he had developed since May 1950 to establish an international research group on European migration and refugee problems. This group, which would eventually become the Research Group of European Migration Problems (REMP) in 1952, initially was a bilateral Dutch-German initiative. Günther Beijer at that moment still was the only formal Dutch representative together with the German Bildungsökonom (education economist) Friedrich Edding, who worked at the Institut für Weltwirtschaft in Kiel. The aims of the research group was to find solutions for the fact that - in their analysis - the relations between population density, the age distribution, the labour potential and the national earning capacity had become so stressed in the second half of the 1940s, that there was a serious risk of social unrest that could endanger prosperity and that could only partially be solved on a national basis. As these Dutch and German researchers were ‘convinced that the integration of the national economies must be accompanied by free migration of labour’, they decided to join forces in order to investigate possible migration problems within Europe. The first studies were published in the spring of 1951 and offered to the US-delegation during the november meeting. Beijer stressed that the idea was to extend this research group ‘by inviting personalities from all nations, who are ready and able, by their scientific qualifications and practical experience, to co-operate on a basis free of national, party political and confessional bias’ (all citations IISG, collection Beijer, inv.nr.33). 

In March 1952 the Research Group was formally established in The Hague. Beijer was appointed secretary and editor-in-chief. The Dutch sociographer Pieter Bouman, professor at Groningen University and personal friend of Government Commissioner Haveman, had been appointed to form the first board of directors. The provisional managing committee consisted of the Utrecht professor and sociographer Sjoerd Groenman and the catholic economist and sociographer professor George Zeegers. In its Mission Statement, REMP highlighted the ‘threat of overpopulation’ for the future prosperity of mankind. Regional unemployment and a falling standard of living could be the results of the disproportional distribution of humanity over the earth. They considered it ‘the imperative duty of scientists and statesmen …to concern themselves with these local disharmonies, by studying them and if possible by indicating solutions to the present difficulties’ (IISG, collectie Beijer, inv.nr. 30). 

 In July 1952 the Research Group organised its first international meeting to discuss its initial plans as expressed in a working paper. As Beijer had been invited by the Americans before the PICMME conference, in return the American Porter Jarrell was invited as an observer on behalf of Pierre Jacobsen, the deputy director of PICMME. He resolutely amended the REMP’s intended research plans, which were primarily focussed on intra-European migration. First he outlined PICMME’s activities: organising migrant transportation via field offices in Germany, Austria, Italy, Trieste and via chartered shipping from the Netherlands to Australia, Brazil, the USA and Venezuela and participating in bilateral agreements. Then he concluded that being so close to the day to day operation, PICMME realised that it might lose sight of the larger demographic implications of its work and therefore looked forward to the scientific studies of the Research Group ‘as possible guideposts for our program in the future’. However, to his view the Research Group was too much confined to intra-European migration only, probably because of ‘a certain distrust of overseas emigration as a cause for the weakening of the basic demographic structure of Europe.’ Jarrell argued that this was not necessary as both sending and receiving states were represented in PICMME. Thus PICMME offered the opportunity to counteract this danger by experimenting with the development of certain flexible controls concerning size, direction and composition of the outflow: ‘If the Research Group were to consider overseas migration it could well recommend to the various governments the nature of the “flexible controls” which might prove scientifically desirable’. The day after Jarrell left the conference Beyer sent him a letter to confirm that the REMP members fully shared Jarrell’s view and would also take overseas emigration into account as an object for study and advice. (all citations IISG, collectie Beyer, inv.nr.31).

It can be concluded not only that the actual founding of REMP and PICMME coincided, but, via the Dutch connection, also that the mutual lobbying and influencing already started during the formative months. The initial discourses concerning labour, population density and demography indicate that the historical actors concerned believed in a makeable welfare society, in which one possible instrument was social engineering via regulating quantity and type of in- and outflow within the respective national populations of the member states. In the Netherlands this was called ‘guided (geleide migratie)’ or ‘planned migration’ (Petersen 1955) and part of the ‘planned modernization’ discourse, as defined by Lutz. However, with Lutz’ remarks on prosopographic evidence in mind, to really substantiate that this expert group in the long run formed a discourse coalition on ‘migration management’ and to be able to discover and explain shifts in the presumed networks of actors and discursive elements, we have to establish who were the key scientists involved, and who were the key political actors. How were they connected to the key political actors and institutions mentioned above? And finally: what were the topics of this discourse and how did they develop? In the next section we reflect on the choices of sources and data and on the methodological choices we had to make to start answering these questions.

## The REMP network

REMP was an informal association of social scientists that is much closer to political policy making (both national and international) than was previously known from the literature. This would make it an informal technocratic association, because of the association and the exchange of knowledge between public administrators and scientists. On the one hand this makes policy (in this case migration management) supposedly more rational because it is based on research and scientific insights. But at the same time, this kind of technocratic associations

- make policy making less transparant and less subject to democratic checking point of view  <b style="color: red;">&lt;"checking point of view" loopt niet. "checks and balances"?&gt;</b> and
- science may be used as a justification of politically motivated motives  <b style="color: red;">&lt;"political motives" of "politically motivated decisions"?&gt;</b>; assuming that the research itself still conforms to scientific standards, it is relevant to compare the choice of subjects with the political agenda.

### Research Questions

Researching the discourse coalition consists of two parts. We first have to make it probable that there were networks that together can be considered to form a discourse coalition involving both researchers and admistrators. After the existence of this association is made likely, we can investigate what research content was produced in the networks. Both REMP and ICEM published journals and other research. To see if and how it was influenced by the REMP/ICEM discourse coalition, we will compare it with contemporary research about migration from a journal that was not associated with either REMP or ICEM. 

To investigate the technocratic nature of the coalition, we study the network of actors, to address the following research questions:

1. Did REMP function as a nexus between public officials and social scientists in the 1950s, and if so, how?
2. ICEM was founded in approximately the same time as REMP. They appear to have gotten closer ties from 1961, when the _REMP bulletin_ became part of the _International Migration_ journal. Was the nexus of REMP science-officials carried over and elaborated in ICEM in the 1960s and beyond?
3. At what time did the connection cease or transform and was there a relation with the change of IC(E)M into IOM?

#### From Concept to Research Strategy

We are trying to establish the existence of a discourse coalition, that is to say, associations between scientists and policy makers who are part of administration. The separation between scientists and public administration is by no means absolute. Discourse coalitions evolve over time as persons changed roles, both within and across the academic and administration sides of the aisle, and their working relationships and associations with each other changed.

A discourse coalition is a concept, an analytical construct that is used to characterize a perceived association between researchers and public administrators based on discursive characteristics. The REMP and ICEM discourse coalition was never a static network, but it evolved over time. Because of this, we have to use different data and datasets for evidence. The evidence for the discourse coalition is collected in two different ways:

1. by close reading archival materials in which the association between public administrators and researchers is mentioned, either implicitly or explicitly, 
2. by measuring the association between researchers and public administrators in one or more networks. 

In this case, we are talking about several networks as migration management is a long running and increasingly international issue that have to face challenges that change over the years. The connection between the public administrators and the scientists can be established if they take part in the activities of the same organisation. In this case, the organisations are REMP and ICEM and the activities we have evidence of are research publications in the form of books, articles and other writings, like supplements. 

The data are localized in time and in scope that we have outlined in the diagram below (see figure 1)

<img src="../images/diagram_1.png">

#### Data sets

Our research materials consist of both types of materials. The archival materials that lend themselves for close reading mainly stem from the Ministery of Social Affairs at the National Archives The Hague and the private collection of Günther Beijer at the International Institute for Social History in Amsterdam. ICEM and its successor IOM are international organizations, of which the archives are essentially closed for researchers. Although a large part of the ICEM-archive of the Dutch representatives can be found in the archival collection of Social Affairs, this is not a correspondence archive and thus not very suited for answering these research questions. As it is, from Beijers collection, which has several files on REMP, it is clear that both REMP and ICEM originated at the same time in the early 1950s and in their inception involved some of the same key people, even if they were separate organizations. But this is not the same as stating that they were part of a joined initiative or project.

We have different long-running datasets that make it possible to establish the overlap between the different organizations involved. Because we are talking about a research-administrative coalition the datasets contain documents from both research and from the administration, the latter being the relevant boards and governments involved with REMP and ICEM. As researchers publish, covering scientific output provides more detailed information per person than just the fact that they occupied an administrative positions.

Below, we have indicated for the datasets which aspect of the discourse network they contain, including the key persons for the network.

**I** - The ministery of Social Affairs and mainly the _Regerings Commissaris voor Emigratie_ (RCE, English: _Government Commissioner for Emigration_), **Bas Haveman** (but also other politicians and administrators) were very much interested in emigration affairs and in founding policy on scientific data, mainly sociology, statistics and demography. (Van Faassen, 2014, esp. pp.8, 231-232, 277-279)

**II**- ICEM and IOM directors and deputy-directors. **Haveman** was one of the directors (1961-1969)
Data derive from Van Faassen, _Polder en emigratie_, 266-277, https://www.iom.int/iom-history; https://www.iom.int/directors-general-and-deputy-directors-general. 

- the ICEM directors were important in international migration management 

**III** - About REMP see above. It was founded in 1952 by **Günter Beijer**. REMP Board membership lists exist from 1952, 1954 and 1969 and a supplement from 1961. The board consisted of scientists **(of which a lot of them also had had or still had jobs in administrative or political advisory positions)**, administrators and some politicians, from the Netherlands and abroad. (see spreadsheet **sheet persoonscategorieen**). This is evidence that these persons were formally associated in the REMP organization in the 1950s until (at least) 1969. **A discourse coalition is a loosely connected network, that can be established by the informal contact in a research group <Kan deze zin niet weg>**. Data derive from the 1954, 1961, 1969 listings in the Beijer Archive (IISG, inv.nr.30) and for the 1952 listing the backcover of REMP publications no. VII (E.W. Hofstee, Some remarks on selective migration, The Hague, Martinus Nijhoff, 1952)

- REMP board members were important in migration research, at first mainly in the Netherlands and Germany.

**IV** - REMP publication series and ICEM-funded series. 


* Studies over NL emigratie
* Studies in Social Life (1953-1974, only migration studies)
* REMP Bulletins Supplements (1954-1984)

The common research interests and associations between the public administrators and the scientistsis can be derived from the several series of publications from REMP. Apart from the authors of the studies themselves, these studies reveal persons in different roles. There were writers of prefaces and introductions and people who funded research that all had different contributions. 

The prefaces and introductions were often written by administrators or politicians who also financed the publication or the research or both, but did not contribute to the content. This is not to say that the research was not independent in a scientific sense, but that there was a direct interest in the subject and that the study was considered beneficial for the process of policy making or evaluation. **Günter Beijer**, who founded REMP, was the editor in chief of the **REMP  and Social Life** series throughout the 1950s, 1960s and 1970s.

- the titles of the collection and the different roles represented in them provide information about the discourse coalition network of REMP from 1953-1984.

**V** - REMP bulletin authors (1952-1962). REMP bulletin was edited by Günther Beijer from 1952. It started out as an informal newsletter but grew into a more serious publication platform for social scientists. 

- the position of Beijer as editor established a network tie with the authors of REMP bulletin

**VI** - In 1961 it merged with the _Migration/Migración_ journal into the ICEM sponsored _International Migration_, that is edited by Wiley. See below for further explanations

- authors and titles of _International Migration_ provide indications for who was active in studying migration

**VII** - As a comparison we use the _Internation Migration Review_, that existed from roughly the same time, but was not related to ICEM (see below for further explanations)

- _Internation Migration Review_ forms a background dataset to compare against the International Migration journal.

The different datasets all contain information about parts of the technocratic network of scientists and administrators who were involved in what we perceive as an evolving discourse coalition about migration and migration management. Each dataset only has data about part of the network but is connected to the other datasets because they overlap in terms of people and time. Together the sets allow for an analysis of the evolving network.

As a consequence, the analysis consists of several parts. 
   - We first establish the existence of a network and a discourse coalition in REMP. This is a network with different roles that we can visualize as nodes, and different types of relations as edges.
   - Then we establish which REMP members were also active in the ICEM network and which other people played an important role there. As the different datasets contain only a part of the network, they either contain information about the academic or the governance activities. It is not useful to visualize this as a node-edge network as many of the relations are not in the dataset. However, the main contribution is to show who was part of which network. Therefore we visualize the networks as overlapping sets of people.

## REMP discourse coalition

For the construction of the REMP network as shown in figure 1, we used the data from the REMP publications (dataset IV), as it contains most detailed data about associations between people. We divided it into three periods: the 1950s, 1960s and 1970s. The 1970s also include the few publications that were published in the 1980s. The heydeys of REMP were in the 1950s, when there were 87 titles in the publications, against 26 in the 1960 and 12 in the 1970s. This is mirrored in the different networks involved, that counted 170 connections (edges) between actors in the 1950s, 62 in the 1960s and 31 in the 1970s.

Over the whole network there were in total 134 different contributors to the volumes - 113 article authors, 23 preface authors, 6 introduction authors, 2 editors and 4 funders. Most (102) made only a single contribution of any sort, 18 made 2 contributions and only 13 contributed 3 or more times. 

Most authors also wrote or co-authored a single article; 26 authors contributed to more than one.  <b style="color: red;">&lt;Hoe verhoudt deze zin zich tot de laatste zin van vorige alinea?&gt;</b> About most authors we do not know much, but all authors we know had an academic status. Of the authors, 8 were active in other roles. REMP authors were not just from the Netherlands. For the 19 authors whose nationality is known, 7 were from the Netherlands, 2 from Germany, France and Australia each and 1 each from Brazil, Israel, Italy, Swiss, Sweden and the United Kingdom (figure 2). 

Some authors also fulfilled other roles in the network. Günter Beijer was the editor of the whole series and 7 wrote a preface for works in which they were not involved as authors. Groenman and Zeegers held three different roles, as they also wrote an introduction.

In [1]:
from scripts.map_visualisation import compound_aut_chart

compound_aut_chart

alt.HConcatChart(...)

There were 25 persons or institutions who either wrote prefaces or funded the REMP publications. ICEM was the largest supporter with 4 titles, there were another 4 financers from the Netherlands and 1 each from Swiss, France and the UK.

This description is partly also visualised in the graphs (figure 2).<b style="color: red;">&lt;Ik neem aan dat de volgende figuur hier ook bij hoort. Hier moet nog wat tekst en uitleg bij, ook bij de tabel hieronder. Er zit ook geen header bijdeze cel, check of narrative OK is.&gt;</b> 


There are a few important conclusions that follow from this analysis. 

1. Günter Beijer was the founder of the REMP and throughout the 1950s, 60 and 70s the central person for its scientific output. His network consisted of academics and administrators that collaborated on a number of studies.
2. The people and organisations in the network were not only from the Netherlands, but also a number of other countries. 
3. There were numerous connections between politicians and academics that together established a discourse coalition, in which the academics (mostly) wrote the contributions while administrators and their organisations supported the research by funding, by commissioning studies and by writing prefaces that pointed out the importance for policy formation.
4. The international organisation ICEM was involved explicitly in the network from 1958 when it started to fund studies in emigration. This was mediated through Bas Haveman, who in 1962 became director of ICEM. 

In [416]:
from scripts.network_visualisation import vconcat
vconcat.configure_view(continuousHeight=1000, continuousWidth=1000)
vconcat

alt.VConcatChart(...)

In [417]:
from scripts.graphs_and_tables import overview
pd.options.display.max_rows = 999

overview.sort_values(['total','authors'], ascending=False)

authors  pref_a  intro_a  \
Beijer,  G.                                              12       0        0   
REMP                                                      0       0        0   
Hofstee,  E.W.                                            4       1        0   
Groenman,  Sj.                                            2       2        1   
Edding,  F.                                               4       0        0   
ICEM                                                      0       0        0   
Richardson,  A.                                           3       0        0   
Radspieler,  T.                                           3       0        0   
Hack,  H.                                                 3       0        0   
Mol,  J.J.                                                3       0        0   
Zubrzycki,  J.                                            3       0        0   
Isaac,  J.                                                2       1        0   
Sauvy,  A.                                                1       2        0   
Zeegers,  G.H.L.                                          1       1        1   
Rubin,  E.                                                2       0        0   
Hofstede,  B.P.                                           2       0        0   
Petersen,  W.                                             2       0        0   
Nixon,  J.W.                                              2       0        0   
Mast, van der W.                                          2       0        0   
Bastos de Avila,  F.                                      2       0        0   
Frijda,  N.H.                                             2       0        0   
Rosenfield,  H.N.                                         2       0        0   
Timlin,  M.F.                                             2       0        0   
Appleyard,  R.T.                                          2       0        0   
Bouscaren,  A.T.                                          2       0        0   
Gadolin, de A.                                            2       0        0   
Wentholt,  R.                                             2       0        0   
Beld, van den C.A.                                        2       0        0   
Lehmkuhler,  M.                                           2       0        0   
Ipsen,  G.                                                1       1        0   
Mill, van A.N.                                            1       1        0   
Haveman,  B.W.                                            0       2        0   
Brink, van den T                                          1       0        0   
Olsson,  B.                                               1       0        0   
Ex,  J.                                                   1       0        0   
Sumiya,  M.                                               1       0        0   
Witcover,  J.                                             1       0        0   
Hyrenius,  H.                                             1       0        0   
Maris,  A.                                                1       0        0   
Mackenroth,  G.                                           1       0        0   
Wilder-Okladek,  F.                                       1       0        0   
Gottmann,  J.                                             1       0        0   
Dorpema,  H.                                              1       0        0   
REMP,                                                     1       0        0   
Glinstra Bleeker, van R.J.P.                              1       0        0   
Kraak,  J.H.                                              1       0        0   
Weinberg,  A.A.                                           1       0        0   
Meyers,  H.G.                                             1       0        0   
Steigenga,  W.                                            1       0        0   
MacKay,  M.C.J.                                     

### Network Overlap

From the time Haveman took office as Government Commissioner and later as Director-General at ICEM, he involved the REMP network in its operations, striving to found policies on research. This was evident when the previously separated journals _Migration_ and the _REMP Bulletin_ were merged into a new journal, _International Migration_ (IM). But also the network itself was extended. The REMP publications dataset is not suited to investigate this. Therefore, we have compared the evidence we have in other datasets for overlap. These datasets are show in the diagram in figure 1. 

More specifically, we study the overlap between members of the REMP Board, the ICEM directors and deputy directors, the Dutch Government and the authors of the studies in REMP, IM and the _International Migration Review_ (IMR) journal. 


In [418]:
import pandas as pd
#from scripts.network_overlap import n_all, n_authors, n_board, n_board_and_author
from scripts.network_overlap import temp_df

n_all = temp_df.author_surname_initial.unique()
n_authors = temp_df.loc[temp_df.in_pub>0].author_surname_initial.unique()
n_board = temp_df.loc[temp_df.in_board>0].author_surname_initial.unique()
n_board_and_author = temp_df.loc[(temp_df.in_board>0) & (temp_df.in_pub>0)].author_surname_initial.unique()

nw_nrs = {'all_network':len(n_all), 
'authors':len(n_authors),
'board':len(n_board), 
'board_and_author':len(n_board_and_author)}

nw_div = pd.Series(nw_nrs).to_frame().rename(columns={0:"number"})
nw_div['%'] = round(nw_div.number/len(n_all)*100, 1)
nw_div

number      %
all_network         2540  100.0
authors             2499   98.4
board                 72    2.8
board_and_author      31    1.2

<b style="color: red;">Import getallen aanpassen</b>

In the datasets, there are 2540 records. Most of the people in the migration managment discourse network (98,4 percent) were authors, while 2.8 percent were on the governing boards of either REMP or ICEM. Only 1.2 percent were active in governance and were authors as well. Our datasets allow us to analyse the evolution of the network over time to a limited extent, as the REMP network decreases in activity after 1970 and as we have much less detailed data about the ICEM network in which the REMP discourse coalition was absorped. The analysis per decade in figure 5 therefore has an emphasis on the 1950s and 1960, when the REMP discourse coalition was most active. REMP founder Günter Beijer was active from the start in 1952 until his death in 1983, but he was followed by other academics and organizers. Australia-based Reginald Appleyard, for instance, appeared in the REMP network in the 1960s, and remained active in the ICEM network in governance and as an academic until the 1990s.

Haveman became ICEM director in the early 1960s, but his predecessors and followers were not Dutch. On the basis of the national provenance of the people in the network of REMP and ICEM it can be concluded that REMP was predominantly (though not exclusively) Dutch oriented in the 1950s, but got more international in the 1960s and even more so in later decades. We have visualized the nationalities of the people in the discourse coalition networks in different decades, as far as we were able to track these down (figure 6). Especially for the later decades, our data about nationalities mostly comprise the administrators in the network, so overall conclusions cannot be very firm. It is clear, however, that the importance of the Netherlands decreased over time and that the network got more international and operated under auspices of ICEM. 

This is mirrored in the scientific output of the discourse coalition, as the REMP publications decreased in number over the years. As we have outlined above, the REMP bulletin merged into the ICEM sponsored _International Migration_ journal. But it was also a result of the changing nature of migration in the Netherlands, that changed from an emigration country in the 1950s and 1960s to a mostly immigration country from the 1970s. The Netherlands shared this change with much of Europe. At the ICEM sponsored ‘Seminar on "The Role of International Migration in the Development of Modern Society” in Turin on 14 and 15 October 1961, the French technocrat and at the time chair of the REMP-board Alfred Sauvy remarked that "[s]ince the discovery of the New World, migrations have traditionally been from Europe, a developed region, to [...] America, Africa and, to a far less extent, to Asia. We are therefore confronted with an unexpected reversal of the migration flow [...] which no one at all had foreseen ten years ago' (preliminaries_2009, 21-22). This was in the years that Haveman was ICEM director and leads us to the discourse part of the migration management discourse network.

In [1]:
from scripts.network_overlap import temp_df as df_board_w_publ, temp2_df as df_decade
from scripts.data_wrangling import highlight_decade

df_decade.sort_values(by='author_surname_initial').style.apply(highlight_decade, axis=1)

<span style="color:blue">Rik: this is what I made from this paragraph. MvF heeft aanvullingen in zwart toegevoegd</span> 


The table in figure 5 contains a list of persons who both published research and had an administrative role. Each person is listed at least twice, once for the decades in which they publish in REMP or IM, once for publications in IMR and once for each administrative role they had. From the 80s, the list of persons with double roles decreases. Also, the balance shifts from Dutch to non-Dutch persons. 

It is tempting to conclude from the decrease of the network that it fell apart and that the discourse coalition disappeared in the 1980s, but we argue that that is not the case. Rather, the explicit involvement of adminstrators in the network became less prominent and the discussion shifted to the academic part of the network. **We know from the Beijer archive that at the end of the 1960s, beginning of the 1970s there was a dip in the REMP/IM activities and that he established contacts with IMR-editors (which is shown in the data, as is for Beijers Australian successor Appleyard)**.  At the same time, policy making strategies **in ICM/IOM migration management** was increasingly handed over to **regional activities in the public domain in collaboration with** think tanks and consultancy agencies **(mostly on border control with biometrical instrument, Cf. Klekowski von Koppenfels 2001, 72, 78-80; Andrijasevic & Walters 2010, 988, esp. note 17.)** that were neither part of academia nor of public administration, and certainly contributed to a new phase in technocracy. They were informed by and participated in the ongoing discourse about migrant management. Because these agencies have no obligation for public accounting, as they perform their studies for political or administrative actors that do no longer directly interact with research, but are indirectly still informed by it. While it became harder to follow the administrative part of the discourse coalition **from the datasets we used**, we can still observe its research part as this continued in the same academic journals as before. 

In [420]:
from scripts.map_visualisation import board_charts
board_charts

alt.VConcatChart(...)

<b style="color: red">Do we need to say more here? E.g. some explicit references to the table and chart?</b> Ja absoluut: eerst uitleg waar de lezer hiernaar kijkt, daarna onze analyse (ter bespreking zometeen).



## Discourse evolution ##


### Discourse Research Questions

The overall subject of the discourse in the discourse coalition was the management of migration. As we wrote above, migration in the early 1950s, when both REMP and ICEM and its predecessors were founded, was mainly about emigration from Europe. In the course of time this changed as Sauvy tentatively remarked in the early 1960s quoted above. **<toevoeging Marijke mbt Sauvy, verwijzing Nord, introductie Third World>**Without going into the details, it is obvious that Sauvy was right, and there was a change in migration streams in which Europe became an immigration area, instead of mainly an emigration area. It is also obvious that these changes in migration would have their influence on the discourse about migration management. In this section, we will analyse this evolution.


In a first attempt to come to an analysis of the evolution of the discourse, we used the Worldcat analysis and word clouds for them (see REMP: https://worldcat.org/identities/lccn-n50070935/
ICEM: https://worldcat.org/identities/lccn-n50055088/
IOM: https://worldcat.org/identities/lccn-no90018047/). However, upon closer inspection and reflection, it is impossible to either know or distinguish who (i.e. which library) added those keywords or at what time. Moreover, it is not clear whether there were conscious or unconscious (or both) changes in the editorial policies for keywords. Therefore, we decided the wordclouds of Worldcat for our purposes should be regarded as artefacts constructed by Worldcat and could not be regarded as reliable qualifications of the discourse and its evolution. 

For our analysis of the discourse we have used other sources, of both a qualitative and a quantitative nature. The main sources we used are the articles in the _International Migration_ (IM) journal published under ICEM (and later IOM) sponsorship (datasource VI in the diagram above). As a comparison, we used the _International Migration Review_ (IMR) journal, with a comparable general migration scope and published in roughly the same period as _International Migration_ but  independent from ICEM. The aim of the comparison is to see whether the academic content of both journals was mainly determined by either developments in migration or those in the migration studies academic field or whether there was a lasting influence of ICEM/IOM sponsorship.

<b style="color:red">In de slack discussie staan nog wat links naar de missies van de twee journals. Die toevoegen? Marijn: ja, plak ze er i.i.g. in, dan kunnen we bespreken en beslissen. De missies ondersteunen ons punt en laat zien dat het niet alleen onze eigen interpretatie is.</b>

In our quantitative analysis, we used the titles of the non-anonymous articles in both journals. <b style="color: red">Is eerder al uitgelegd wat we bedoelen met die anonymous articles? Authorless articles (in the tables of content) do not contribute to the networks, but they are very much part of the discourse.</b> We did not use the article fulltext, as we had no access to fulltext, but also because in our opinion the titles can be regarded as a means to engage in discourses, as they are the first and primary means to attract the attention of the readers of the journal. 
For the qualitative analysis, we used the anonymous articles in the journals, under the assumption that anonymous articles were meant as a collective statement. Upon closer inspection, it appeared that this was indeed the case, as apart from review articles, bibliographies and a single obituray, they were mostly reports from ICEM conventions, that were held irregularly over the years. The close reading of these editorial articles offers an opportunity to explore the way ICEM conceptualized the relationship between science and governance. Therefore we start by outlining these discussions and use a quantitative analysis of the journal titles to confirm or amend this qualitative analysis.

Table X: article types for a sample of the anonymous articles published in IM. The types are based on the title of the article or when no title is provided, on the section in the journal in which it appeared.

| type of article | number |
|-----------------------------|--------------|
| Editorials (+Errata) | 7    | 
| Activities of REMP       |         3|
| Book (reviews / periodicals)    |        19|
| In Memoriam           |         2|
| Institutes (newly est.)        |        3|
| | |
| Conferences (1952 (2x); 1956, 1961) |   4 |           
| Statistical Reports (1957-1960)  |      6|

### Seminars ###

In 1982 it was stated that “The Intergovernmental Committee for Migration (ICM) has organized Seminars on Migrant Adaptation and Integration regularly since 1974.” but there were previous conferences that had a similar goal. In 1962 Haveman wrote that in 1961 there had been a ‘Seminar on "The Role of International Migration in the Development of Modern Society” It took place in Turin on 14 and 15 October I961 and was organized by the Intergovernmental Committee for European Migration, with the co-operation and assistance of the Italian authorities.’  Contrary to later seminars, it included exclusively academic speakers and did not end in concrete policy recommendations. The 1964 “Icem - XXI Council Session” as it was rather vaguely called in the 1964 editorial was concerned mainly with Latin America. The council session had speakers that were country representatives and no what elsewhere are consistently referred to as ‘experts’. <b style="color: red">&lt;Laatste deel van de zin is moeilijk te parseren. Iets als "but no, what elsewhere are consistently referred to as, 'experts'" MvF Hier dieper de analyse in: het editorial refereert dus puur een een Zitting van de ICEM-raad...kortom: politiek in het tijdschrift, want de ICEM onder Haveman had Latijns-Amerika op de agenda + literatuur verwijzingen bio Haveman.&gt;</b>

There is a gap from 1964 to 1975 in these political documents <b style="color: red">&lt;voor de zekerheid: de PDFs die ik in de surfdrive heb gezet zijn een sample, dus er zijn er veel meer, ook voor de jaren 1964-1975.&gt;</b>, but the purpose of the seminars is very clearly stated in 1979 “The ICEM Seminars on Adaptation and Integration of Permanent Immigrants are intended as a forum for an exchange of views and experience in the field of migrant adaptation and integration between representatives of governments and international governmental and non-governmental organizations concerned with migration. The Seminars usually deal with two main subjects, divided into sub-topics, i.e. individual aspects, which are analysed and discussed on the basis of working papers prepared by experts on the subject matters chosen. In addition, governments and international organizations contribute information documents reflecting their own experiences and/or policies in those fields. General and specific recommendations on each sub-topic are formulated by the participants for guidance and action on a national or international level respectively.”  And it also gives an insight into the preparation of a Seminar: “The programme for this Seminar was prepared by a Working Group under the chairmanship of Mr. Rent Fourdin, Director General of the Chancellery, Ministry of Foreign Affairs, Belgium. The Working Group was composed of the Government Representatives of Argentina, Australia, Belgium, Brazil, Canada, Israel, Italy, Portugal, Turkey and the United States of America. [...] Experts from Australia, Belgium, France, Italy, Portugal, the Netherlands, Spain,Switzerland, Turkey and the U.S.A. prepared 11 working documents on the above-mentioned subjects and acted as discussion leaders during the Seminar.” 

<b style="color: red">&lt; Referentie naar het anonynieme artikel toevoegen: Introduction, pp. 7-12, Volume 17, Issue 1-2, January‐April 1979, DOI: https://doi.org/10.1111/j.1468-2435.1979.tb00844.x. &gt;</b>

After that, this is repeated several times and the Seminar also recommends ICEM to regularly organize seminars. 

1. “The Seminar recognizes the very considerable efforts that the ICEM Administration has to make in the organizing of Seminars. It stresses the importance of this international forum for the discussion of problems concerning migrants’ integration and submits to the competent governing bodies of ICEM the suggestion that the organization of seminars become a regular function of the Committee. 
2. The Seminar expresses the wish that at the appropriate time, a fifth seminar be organized under the auspices of ICEM and that the mandate of the Working Group which has assisted in the past in the preparation of seminars be confirmed. The principle of rotation among its members should be maintained.”

<b style="color: red">&lt;General Conclusions/Conclusions Générales/Conclusiones Generales, pp. 13-18, Volume 17, Issue 1-2, January‐April 1979, DOI: https://doi.org/10.1111/j.1468-2435.1979.tb00845.x &gt;</b>


### About discourse coalitions ###

The desirability of a close connection between research and governance is affirmed over and over. The first explicit mention is in 1963, when the editorial stated that “Thanks to [a growing number of migration]  studies, a greater knowledge has been acquired of the many aspects of the migration process and thus it has been possible for the immigration countries to establish the conditions which will allow their immigration policies to succeed.” At the time much attention was devoted to migration in Latin America. The second 1963 editorial referred to ICEM policies regarding agricultural productivity: “Many of the programmes which the Intergovernmental Committee for European Migration has established to assist its Latin-American Member Governments are precisely designed to help with these problems. ICEM’S policy in the agricultural field is therefore designed to select and resettle small groups of skilled European farmers trained in up to date techniques of agricultural production.”  <b style="color: red;">&lt;Referenties naar anonieme artikelen toevoegen. &gt;</b>

An explicit connection between research and governance was made in 1975 when in the recommendations of the second ICEM Seminar (which from then on included representatives from governments, international and non-governmental organizations as well as experts) the first point was: “The establishment of a body of statistical data could be instrumental in coping with the range of problems concerning migration and more particularly family migration and/or family reunion. Such data are seen as an essential element also at the policy making level. There is therefore a great need for research both at the national and at the international level and governments are urged to take appropriate measures to make such research possible. Methods of data collection should not, however, make more difficult or increasingly hamper the movement of people across frontiers.” After that, the need for better statistics for policy making was explicitly repeated in all the Seminar recommendations in almost the same words. And in 1983 the Seminar even recommended that “ICM, in collaboration with other organizations, should be encouraged to establish pilot case studies to research this phenomenon. These could serve as an input for regional discussions.” <b style="color: red;">&lt;Referenties naar anonieme artikelen toevoegen. &gt;</b>

It is remarkable that there is only an explicit call for statistical research. When in 1979 the topic of the Seminar was the adaptation of child migrants, the Seminar concluded that “scientific analysis and research in the field of the socio-psychology of the children of migrants has produced a whole range of very divergent data.” and that “There was a strong feeling that the results of this research are badly in need of being integrated and collated.” But immediately following this, the Seminar warned that “considering that the socialization problems of migrant children in the host country are conditioned by their economic, social and legal status in that country, the Seminar has drawn attention to the dangers of an excessive intrusion of psychology into the problems of the children of migrants.” This is the only instance when another field of research than statistics or demography was mentioned. <b style="color: red;">&lt;Referenties naar anonieme artikelen toevoegen. &gt;</b>

### Agenda setting


From the first seminar, these seem to have had a strong (research) agenda setting purpose. Haveman stated that “The fundamental problem facing the world today is how not to hinder, indeed how to facilitate, the development of society insofar as is considered to be advantageous to its members [...] Does international migration contribute to the solution of this problem; if so, what type of migration and in what way ?” We already cited Sauvy who stated around the same time that from the beginning of colonization, it had led to a migration from Europe, as a result of overpopulation but that at the beginning of the 1960s that was possibly changin. 

From the 1963 issue, however, attention was directed for a while to migration to Latin America: ”In order that our readers may be better acquainted with some of the problems facing Latin America today and with their many implications we have, therefore, decided to devote most of this issue of International Migration to the study of various agricultural settlements in Latin America” and in the following seminar US representative “Abba P. Schwartz said [...] 'Whatever help we give Latin America is, in the final analysis, self-help. Our destinies are bound together. If we are all to survive as free nations, we will only do so together'”, bringing cold war rhetoric directly into the discussion.

There is a gap to the mid-1970s in this kind of reports <b style="color: red">&lt;Zoals ik hierboven al aangaf, er zijn wel degelijk anonieme artikelen in die periode. Is het zinnig als ik er daar een aantal van toevoeg aan de Surfdrive?&gt;</b>, but then research had very different topics and in 1979 “The Seminar, meeting the wish expressed by several Latin American delegations, agreed that the next ICEM Seminar should examine problems more directly connected with that continent.” <b style="color: red">&lt;ja, ook overleggen of in de bovenstaande alinea's al analoge analyse wordt toegevoegd, nl terugkoppeling naar 'worldorganizationconcept dat hiervoor volledig opgaat' en info dat Haveman Latijns Amerika als agenda had vanaf zijn aantreden. Anders bepalen waar we een concluderende alinea toevoegen&gt;</b>


<b style="color: red">Concluding paragraph</b>

## Quantiative Analysis of the Discourse

## Research Questions - discourse

The two separate questions have to be researched in different ways. Research question 1 refers to the content of the discourse and may be broken down in some more specific questions:

* Are there global consistencies in the use of terminology and subjects of the IM journal
* What are the most important topics
* Do the topics shift over time and can we attach a periodization to the shifts. 
    - Does this periodization coincide with periodizations in institutional priorities identified elsewhere or do they either predecede or follow them?
* Are there specific regions featured in the research
    - Does this change over time and if so, is this related to/coincide with shifts in topics

From the _network perspective_ we can add the following questions
* Who are the authors of the articles in IM
* Are specific authors connected to specific topics or regions or both 
    - Are there subgroups of authors concerned with a specific topic. Do these groups change in composition over time
    - Can periodizations and changes in topics/regions be related to specific authors or groups of authors
    

## Research Questions - research-policy connections

* have we been able to establish a connection between the authors and public administrators in our network analysis and does any changes in content reflect these connections
* are there articles that reflect 
* are there any specific articles related to policy issues and how does this manifest itself
* does the periodizations support conclusions about agenda setting on either policy or research side

###  The datasets

The datasets used for the analysis (IV, V, VI and VII in the diagram above) contain a total of 4,282 article titles, divided over 762 _REMP_ and _International Migration_ (IM) titles and 1678 _International Migration Review_ (IMR) titles. IMR also contains 1842 reviews of literature, that we left out of the comparison. The total number of articles increased from 100 in the 1950s to 1,490 in the 1990s. The IM set is smaller than the IMR set, but the rate varies. In the 1950s we have no IM and IMR titles and only 100 for REMP, so trends for this decade are tentative. In the 1960s the rate between IM and IMR was 0.84 and after that it changed to 0.30 in the 1970 to approximately 0.5 in the 1980s and 1990s  <b style="color: red;">&lt;Need to explain what we mean by rate. Is it the ratio of article published per period. I.e. IM published 0.84 articles per 1 article in IMR?&gt;</b>. IMR research is dominant in the total numbers, but we use IMR mainly to see how trends in IM differ from the more general trends, so therefore we will often present a graph of both IM and general (IM+IMR) developments. 

Tracing topics in literature is difficult and subject to interpretation issues in every way. For trend analysis the relatively small word set made using with word counts good enough for our purposes; the method has the advantage of being transparent. Two thirds of individual words in titles only occurred once, so we classified title words into somewhat bigger topics by hand; the classi. Below, our analysis refers to these topics.  <b style="color: red;">&lt;add a note about how we defined the topics. We manually identified topics and constructed word lists per topic by searching through the titles, inspired by the topic categories defined in the UCREL Semantic Analysis System (Wilson, A. (1993). Towards an Integration of Content Analysis and Discourse Analysis: The Automatic Linkage of Key Relations in Text. UCREL Technical Paper 3, Linguistics Department, Lancaster University. Dawn Archer, Andrew Wilson, Paul Rayson (2002). Introduction to the USAS category system. Benedict project report, October 2002). One set of topics covers directions of migration, another set focuses on aspects of management, a third on forms of migration and a fourth set on academic discplines.&gt;</b> The graphs were all made by calculating the fraction of all title words that belong to a specific topic, for five year periods, starting from 1950 until 2000, increasing the start and end of the period by one year in each step (1950-1954, 1951-1955, 1952-1956, etc).

In [2]:
import scripts.journal_analysis as ja

/data/home/jupyter-jdh-artikel/.local/lib/python3.7/site-packages/sklearn/utils/deprecation.py:87: FutureWarning: Function get_feature_names is deprecated; get_feature_names is deprecated in 1.0 and will be removed in 1.2. Please use get_feature_names_out instead.
  warnings.warn(msg, category=FutureWarning)


In [5]:
pd.DataFrame(ja.topic_words.items())[1].apply(pd.Series).fillna('')

0            1               2                     3   \
0         cause       causal     determinant                factor   
1       process      dynamic            rate           development   
2        policy     policies         politic              decision   
3        labour        labor          worker                  work   
4     low-skill      skilled           skill            high-level   
5       illegal        legal    undocumented              unwanted   
6   involuntary       forced          refuge               refugee   
7     immigrant    immigrate     immigration                         
8      emigrant     emigrate      emigration  overseas destination   
9     migratory      migrant       migración             migrantes   
10       family    household       community                 group   
11     identity  nationality     nationalism              xenophob   
12    psycholog                                                      
13     economic        trade                                         
14      statist                                                      
15    sociology       social  socio-economic          anthropology   
16   demography  demographic                                         

                  4             5          6                 7             8   \
0            influen       setting     effect            affect        impact   
1            pattern        change   changing          interact      relation   
2         management      managing     govern         promotion       planned   
3         employment   occupation,                                              
4       professional  intellectual  scientist       brain drain    vocational   
5        undesirable   citizenship      right                                   
6          necessity           war   conflict            asylum                 
7                                                                               
8   voluntary return                                                            
9         migratoria   migraciones    migraci  arbeitsmigranten     migratori   
10                                                                              
11            ethnic          race    ideolog          naturali  assimilation   
12                                                                              
13                                                                              
14                                                                              
15           culture                                                            
16                                                                              

                     9   ...      21     22      23             24        25  \
0           consequence  ...                                                   
1             evolution  ...                                                   
2           recruitment  ...  refine  revis  border  international  national   
3                        ...                                                   
4   vocational training  ...                                                   
5                        ...                                                   
6                        ...                                                   
7                        ...                                                   
8                        ...                                                   
9            migrazioni  ...                                                   
10                       ...                                                   
11          integration  ...                                                   
12                       ...                                                   
13                       ...                                                   
14                       ...                                                   
15           

In [6]:
frm = ja.df[ja.df.continent.notna()].groupby(['issue_decade', 'continent']).agg('count').article_title
xfrm = frm/frm.groupby('issue_decade').agg('sum')*100
ffrm = pd.DataFrame(xfrm)
df_continent_decade = ffrm.reset_index()


In [9]:
c1 = ja.alt_topic_chart(topicwords=['migration','immigration','emigration'],
                   topicdf=ja.df_remp_im, 
                   var_name='migration_type', value_name='number', id_vars='index', title='Migration types in IM')
c2 = ja.alt_topic_chart(topicwords=['migration','immigration','emigration'],
                   topicdf=ja.df_imr_research, 
                   var_name='migration_type', value_name='number', id_vars='index', title='General use of Migration Types')
alt.vconcat(c1, c2).properties(
    title='Migration types'
).configure_title(
    fontSize=20,
    font='Verdana',
    align='center',
    color='black'
)

alt.VConcatChart(...)

In a corpus like this, it comes as no surprise that the most common terms include (variants of) "migration". As figures 7 (terms in IM) and 8 (total terms) show, _emigration_ was more common in the 1960s, while _immigration_ was used more often later on, although there was a concentration in the mid-1960s. The term _migration_ was not used very frequently throughout the whole period, though its use increased slightly over the years. Of course, the terms _emigration_ and _immigration_ depend on the perspective, because what is immigration for one country is emigration for another. However, in general immigration causes more and more lasting changes. In our qualitative analysis of the discourse above, we have already seen how researchers and politicians used emigration to mean European emigration, while immigration (or migration) was the term commonly used for other contexts, although it was increasingly used for Europe as well. The quantitative title analysis of the titles confirms this.

In [10]:
xc = alt.Chart(df_continent_decade).mark_bar().encode(
    alt.Y("article_title:Q", title='number'),
    alt.X('continent:N', title=''),
    color='continent:N',
    column=alt.Column('issue_decade:N', title='Decade')
).properties(
    width=200,
    height=200
).repeat(
    row=['issue_decade:N']
).interactive()
xc.properties(
    title='Continents in titles per decade'
).configure_title(
    fontSize=20,
    font='Verdana',
    align='center',
    color='black',
)

alt.RepeatChart(...)

When we look at the (relative) frequency with which continents are mentioned in the article titles, this confirmes the general trend. In the 1950s the emphasis is mainly on Europe, from where most emigration took place, in the 1960s there was a relative emphasis on Oceania (mainly Australia). In the decades after that, it was mostly on other continents, even if Europe remained important. South America and Africa rarely were the focus in migration discussions. 

In [11]:
c3 = ja.alt_topic_chart(topicwords=['cause_effect', 'process', 'decision making'],
                   topicdf=ja.df_remp_im, 
                   var_name='migration_type', value_name='number', id_vars='index', title='process words'
)

c4 = ja.alt_topic_chart(topicwords=['cause_effect', 'process', 'decision making'],
                   topicdf=ja.df_imr_research, 
                   var_name='migration_type', value_name='number', id_vars='index', title='process words')

alt.vconcat(c3, c4).properties(
    title='Migration Management'
).configure_title(
    fontSize=20,
    font='Verdana',
    align='center',
    color='black',
)


alt.VConcatChart(...)

As this is a discourse coalition about migration management, the next aspect to consider is the extent to which management words played a role in the discourse. Management has a changing vocabulary, but we tried to single out the most important terms. From figure 10 it appears that overall there was a growing interest in management related issues, but overall in IMR discussions about migration management were more important all the time  <b style="color: red;">&lt;Not sure I follow, management is more important than what? Or is it that it is more important in IMR than in REMP/IM? Based on the fractions, decision making covers a higher fraction of IMR titles than IM titles&gt;</b>. This confirms our idea that the ICEM liaison of IM had an influence on its discussions. The topics that the titles referred to shifted from a process to a decision making orientation, in both journals but more so in IM. Without further research, it is hard make any definitive interpretions of this shift, but it seems to be indicative for a shift from an approach of migration management as interacting with migration to a more control oriented approach.

In [12]:
topic_words = ['labour', 'skill', 'legal', 'forced']
c5 = ja.alt_topic_chart(topicwords=topic_words,
                   topicdf=ja.df_remp_im, 
                   var_name='migration_type', value_name='number', id_vars='index', title='process words'
)

c6 = ja.alt_topic_chart(topicwords=topic_words,
                   topicdf=ja.df_imr_research, 
                   var_name='migration_type', value_name='number', id_vars='index', title='process words')

alt.vconcat(c5, c6).properties(
    title='Labour Migration terminology'
).configure_title(
    fontSize=20,
    font='Verdana',
    align='center',
    color='black',
)

alt.VConcatChart(...)

Taken as a whole, there no remarkable difference in the content of the articles of IM and IMR. Over the years the relative proportion of articles that cover a specific topic drops, which may be a sign that the diversity of topics increases, or that terminology diversifies. As an example we have taken the topic of labour migration. Labour in relation to migration was important from the start in both IM and IMR and always remained so. The varying trends in both journals does show that there were dynamics in the scientific discussions in both journals that did not coincide, but without close reading it is impossible to give any more specifics about them (figure 11). 

In [13]:
topic_words = list(ja.discipline_words)
c7 = ja.alt_topic_chart(topicwords=topic_words,
                   topicdf=ja.df_remp_im, 
                   var_name='migration_type', value_name='number', id_vars='index', title='disciplines', scheme='category20b'
)

c8 = ja.alt_topic_chart(topicwords=topic_words,
                   topicdf=ja.df_imr_research, 
                   var_name='migration_type', value_name='number', id_vars='index', title='disciplines', scheme='category20b')

alt.vconcat(c7, c8).properties(
    title='Disciplines in titles'
).configure_title(
    fontSize=20,
    font='Verdana',
    align='center',
    color='black',
)

alt.VConcatChart(...)

However, there is another important difference between IM and IMR in the disciplines that were represented in the articles. Both journals are general social science migration journals without any specific disciplinary orientation. In both IM and IMR sociology was important. In the early years economics dominated titles in IMR, but after the late 1950s this subsided and many disciplines were represented, with some emphasis on pyschology in the 1980s.  <b style="color: red;">&lt;The top figure is REMP/IM, so the dominance of economics is in REMP, not in IMR.&gt;</b> Demography was never important in IMR, but in IM it was always present, while in IMR statistics were always present, and in IM it was only present as a separate subject in the 1980s. In IM psychology got a lot of attention in the late 1960s and early 1970s, but after that psychology was much less of a topic. 

This overview presents some trends, that confirm the general social science nature of both journals, though mainly the social sciences that make heavy use of statistics and statistical methods. Anthropology, for instance, is not part of the discussions in either IM or IMR, even if there is much anthropological research about migration and its impact on migrant groups. The disciplinary topics are, of course, not sharpely separated. For instance, demographics and statistics are closely related, while sociology and economics provide much of the theoretical framework for other social sciences. However, some of the analysis of the qualitative discussion sheds some light on this quantitative approach. As we have seen, up until the 1970s in IM there was a lasting presence of demographers and statistically oriented researchers that in the ICEM conferences persistently called for funding of statistical research, while they then cut short the influence of psychological research, especially in the report of the 1979 conference report. Even before that psychology was not prominent as a topic in the titles anymore.

### Conclusion

The discourse coalition of migration management came into being in the early 1950s when in the Netherlands Günter Beijer founded the Research Group for European Migration Problems and at the same time precursors of ICEM were founded as an international working group for the transportation of migrants. These included first mainly refugees, but later on also European emigrants and still later on diverse groups of migrants. REMP had its roots in the planning-oriented governance circles in the Netherlands, Europe and the World and from its inception included both international researchers and administrators. The aim was to control migration by founding it on research and scientific insights, mainly demography and sociology, due to the orientation of the researchers involved. 

From the late 1950s throughout Europe there was less orientation on emigration and in the 1960s the discourse coalition altered. Orientation became more international and research became more closely allied with ICEM when Bas Haveman became its director and the ICEM supported journal _International Migration_ became the main venue for research. The research group became still more internationaly oriented during Beijer's long dominant position. The researchers were the think tank for ICEM as well, but migration changed when increasing labour migrants started coming to Europe and the numbers of European emigrants decreased. Still, in ICEM governance circles and its associated research group, there remained a perceived need for the management of migration based on insights from research. Notwithstanding the changes in migration, research orientations remained by and large the same and predominantly oriented to statistically founded social science. Growing influences from other social sciences, mainly psychology, were largely cut short in the late 1970s. The control set of IMR shares many of the same orientations, but there seems to be more room for other voices, mainly a lasting interest in psychology. The calls in the conference reports for funding for statistical research give the impression of a group that was striving to maintain its position. 

The discourse coalition that came into being in the 1950s proved to found a lasting relation between governance of migration and the research on which it founded itself. While migration itself changed constantly, the basic orientation of both research and governance remained remarkably constant in a focus on the migration streams and the control of them and much less concerned with either the motivations behind migration or the effects on the migrants. The discourse coalition therefore evolved over time, but remained constant in its conception about the relation between research and governance. This is not to say that research was not independent, but the result was that it was hard to influence the discourse from outside the coalition.

- Terugkomen op Petersen en conceptuele kant. 
- data scopes als lens op hoe we gewerkt hebben, en waar we op stuitten: hoe ver moet je gaan met verantwoording?


## Notes

[1]

[*] Sources differ in their spelling of the name Günther Beijer, using either Beijer or Beyer. We use Beijer throughout the article and the underlying data. 

[2]

